Branch-SDK/src/main/java/io/branch/referral/Branch.java

```diff
@@ -325,19 +340,27 @@ private Branch(@NonNull Context context) {
         deviceInfo_ = new DeviceInfo(context);
         branchPluginSupport_ = new BranchPluginSupport(context);
         branchQRCodeCache_ = new BranchQRCodeCache(context);
-        requestQueue_ = ServerRequestQueue.getInstance(context);
+        requestQueue_ = BranchRequestQueueAdapter.getInstance(context);
     }
 
     /**
      * <p>Singleton method to return the pre-initialised object of the type {@link Branch}.
-     * Make sure your app is instantiating {@link BranchApp} before calling this method
+     * Make sure your app is instantiating Branch before calling this method
      * or you have created an instance of Branch already by calling getInstance(Context ctx).</p>
      *
      * @return An initialised singleton {@link Branch} object
      */
     synchronized public static Branch getInstance() {
         if (branchReferral_ == null) {
-            BranchLogger.v("Branch instance is not created yet. Make sure you call getAutoInstance(Context).");
+            BranchLogger.v("Branch instance is not created yet. Make sure you call getInstance(Context).");
+        }
+        return branchReferral_;
+    }
+
+    synchronized public static Branch getInstance(@NonNull Context context) {
```

Comment:
Can you rename from `getInstance` -> `init`.

`getInstance` is already exists and `init` is more clear as to the purpose.

So called would invoke
`Branch.init()`

And client code would be more readable in my opinion.